ntp-proto/src/filter.rs

```diff
@@ -14,6 +14,9 @@ use crate::{packet::NtpLeapIndicator, NtpDuration, NtpHeader, NtpTimestamp};
 const MAX_STRATUM: u8 = 16;
 const MAX_DISTANCE: NtpDuration = NtpDuration::ONE;
 
+// TODO this should be 4 in production?!
+const NSANE: usize = 1;
```

Comment:
This is a very opaque variable name. Maybe expand the name and/or provide a comment about the semantics?